src/components/ui/CheckboxGroup/stories/CheckboxGroup.stories.tsx

```
@@ -0,0 +1,49 @@
+    return (
+        <SandboxEditor>
+            <CheckboxGroup.Root
+                className="flex gap-4"
+                name="fruits"
+                value={checked}
+                onValueChange={handleChange}
+            >
+                {options.map((option) => (
+                    <>
+
+                        <CheckboxGroup.Item value={option.value}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)